charm/user/schema.py

```diff
@@ -89,30 +89,32 @@ class Arguments:
         email = graphene.String(required=False)
         is_active = graphene.Boolean(required=False)
 
-    def mutation(self, user_id, first_name, last_name, city, country, address, postal_code, telephone, email, is_active):
+    def mutate(self, info, user_id, **_kwargs):
```

Comment:
Please add commentary about the functionality of this mutation.

Reply:
ok.